Game/Resources/VoltageScene/Scripts/RoundAboutMinecartController.as

```diff
@@ -3,6 +3,7 @@ class RoundAboutMinecartController {
     Entity @self;
     Entity @minecart;
     Entity @brakeTarget;
+    Entity @mastermind;
     int phase = 0;
     float speed = 2.0f;
     // v(t) = a*t*t + b*t + c
```

Comment:
Remove comments that's not used.

Reply:
These comments refer to the upcoming variables. It's an explanation so that we know what those variables do and where they come from.